safe_transaction_service/history/management/commands/setup_service.py

```diff
@@ -645,3 +647,30 @@ def _setup_safe_proxy_factories(
                     "tx_block_number": initial_block_number,
                 },
             )
+
+    def _setup_erc20_indexing(self) -> bool:
+        """
+        Update ERC20/721 indexing status if `indexing block number` is less
+        than `Master copies` block deployments, as it sounds like a configuration error
+
+        :return: `True` if updated, `False` otherwise
```

Comment:
This is going to keep always ERC20 indexing behind MasterCopies indexing?

Reply:
No, this is going to make sure that ERC20 indexing is never behind the first created master copy, as Safes cannot exist